### ISSUE TITLE
logits is an undefined name in this context, should it be logits_last?

### ISSUE THREAD

**Issue by cclauss** [astorfi/TensorFlow-World#19]
Undefined names can raise `NameError`at runtime.

https://travis-ci.org/astorfi/TensorFlow-World/jobs/272817788#L623-L626

https://github.com/astorfi/TensorFlow-World/blob/master/codes/3-neural_networks/multi-layer-perceptron/code/test_classifier.py#L113

**Comment by astorfi** [comment]
@cclauss Thank you for your consideration.

**Comment by astorfi** [comment]
It's been resolved!
